Sources/GRPC/AsyncAwaitSupport/GRPCAsyncRequestStream.swift

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if compiler(>=5.5)
+
+/// This is currently a wrapper around AsyncThrowingStream because we want to be
+/// able to swap out the implementation for something else in the future.
+@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
+public struct GRPCAsyncRequestStream<Element>: AsyncSequence {
+  @usableFromInline
+  internal typealias WrappedStream = AsyncThrowingStream<Element, Error>
+
+  @usableFromInline
+  internal let stream: WrappedStream
```

Comment:
If something would be `private` but can't be because it requires a `@usableFromInline`/`@inlinable` then by convention we prefix the name with an underscore.

Reply:
Does this go for type aliases and functions too?

Reply:
I did `stream` and `WrappedStream` in 4fa1c3a2.